data/mods/Magiclysm/items/recipe_books.json

```diff
@@ -18,6 +18,25 @@
     "time": "32 m",
     "fun": 1
   },
+  {
+    "id": "black_dragons_historical",
+    "type": "BOOK",
+    "name": { "str": "A Comprehensive Analysis on Black Dragons", "str_pl": "copies of A Comprehensive Analysis on Black Dragons" },
+    "description": "This book painstakingly delves into the history of black dragons, gruesomely detailing their acts, the cults that worship them, and those that oppose them.  In its pages are some alchemical formulae that can be concocted using the remains of dead black dragons.",
+    "weight": "1 kg",
+    "volume": "750 ml",
+    "price": 7900,
+    "bashing": 5,
+    "material": [ "paper" ],
+    "symbol": "?",
+    "color": "black_white",
+    "skill": "spellcraft",
```

Comment:
i don't want books to train spellcraft outright; i want spellcraft to be a result of studying spells.

Reply:
I pushed an update that changes the "required_level" to the same as the "max_level", though I'm not sure if I did that correctly. I see the commit in the PR, but if it would be better I can include it as a suggestion for you to review and then commit that change.